src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/Index.rst

```diff
@@ -27,5 +27,7 @@ Wellbore Problems
 
    casedThermoElasticWellbore/Example
 
+   thermoPoroElasticWellbore/Example
```

Comment:
This example belongs to both `wellboreProblems` and `thermoPoromechanics`. For instant it is placed in `wellboreProblems` because all the wellbore related problems are placed in that location.